RecoLocalTracker/SiStripClusterizer/interface/StripClusterizerAlgorithm.h

```diff
@@ -24,25 +24,28 @@ class StripClusterizerAlgorithm {
   // state of detID
   struct Det {
     bool valid() const { return ind != invalidI; }
-    float noise(const uint16_t& strip) const { return SiStripNoises::getNoise(strip, noiseRange); }
-    float gain(const uint16_t& strip) const { return SiStripGain::getStripGain(strip, gainRange); }
-    bool bad(const uint16_t& strip) const { return quality->IsStripBad(qualityRange, strip); }
+    uint16_t rawNoise(const uint16_t strip) const { return SiStripNoises::getRawNoise(strip, noiseRange); }
+    float noise(const uint16_t strip) const { return SiStripNoises::getNoise(strip, noiseRange); }
+    float weight(const uint16_t strip) const { return m_weight[strip / 128]; }
```

Comment:
shouldn't this be ```aveWeight``` to denote that this is per channel?
Well, looking at
https://github.com/cms-sw/cmssw/blob/4c0913fabe8081975cee9b0683d6906338a7da3f/CondFormats/SiStripObjects/interface/SiStripApvGain.h#L78-L81
it was already the case in a less explicit way.
So, this name change will make it more explicit.